src/web100srv.c

```diff
@@ -878,7 +680,7 @@ int run_test(tcp_stat_agent *agent, Connection *ctl, TestOptions *testopt,
   /*  alarm(15); */
   // Run scheduled test. Log error code if necessary
   log_println(6, "Starting middlebox test");
-  if ((ret = test_mid(ctl, agent, &*testopt, conn_options, &s2c2spd)) != 0) {
```

Comment:
Ack.  (I'm responding to or acking every comment, because every unreplied comment is a todo item when I scan the pull request)